packages/core/src/app/common/error/SentryErrorLogger.spec.ts

```diff
@@ -154,45 +161,13 @@ describe('SentryErrorLogger', () => {
 
         expect(init)
             .toHaveBeenCalledWith(expect.objectContaining({
-                blacklistUrls: [
+                denyUrls: [
                     'polyfill~checkout',
                     'sentry~checkout',
                 ],
             }));
     });
 
-    it('does not rewrite filename of error frames if it does match with public path', () => {
```

Comment:
The rewriting will only happen when "the frame filename begins with `/` or a Windows-style prefix such as `C:\`", hence this test is no longer relevant.
https://github.com/getsentry/sentry-javascript/blob/487810a32d6d6c5356386ea0b55b84a85675cb04/packages/integrations/src/rewriteframes.ts#L70

Reply:
I think we still need it, as the same check exists in [5.27](https://github.com/getsentry/sentry-javascript/blob/250063170e0c871d52a83a414aeff7229857a28c/packages/integrations/src/rewriteframes.ts#L64). The test is intended to verify that this handler works as expected - https://github.com/bigcommerce/checkout-js/blob/3484d2aa1ab7ac131f089b90185dc4aff53d956e/packages/core/src/app/common/error/SentryErrorLogger.ts#L138 IIRC, due to the check mentioned above, the Sentry client doesn't automatically rewrite the frames and add the `app://` prefix so we got to do it manually ourselves.

Reply:
Thanks @davidchin . I have restored it.
Just one more concern, the assertion of this test seems like always true. 
Did I miss anything?
```
const frame = output.exception!.values![0].stacktrace!.frames![0];

        expect(frame)
            .toEqual(frame);
```

Reply:
Hrm good catch @bc-peng, could you please change 
```ts
        expect(frame)
            .toEqual(frame);
```
to
```ts
        expect(frame)
            .toEqual({
                ...frame,
                filename: 'https://cdn.hello.world/js/app-123.js',
            });
```

Reply:
Hi @davidchin, I have updated `handleRewriteFrame` as well in [the new commit](https://github.com/bigcommerce/checkout-js/pull/966/commits/5100ee0f896361c4f78ff0968600dad1f4a41232).

In this test, `does not rewrite filename of error frames if it does match with public path`, the filename shares the same host as the public path.
In another test  few lines up, `configures client to rewrite filename of error frames`, the filename is on a different host.

Reply:
For that test, you need to keep the file name to be `https://cdn.hello.world/js/app-123.js` for both the input and the expected output.

Reply:
Done. Did I get it right? 🙇🏻‍♂️
https://github.com/bigcommerce/checkout-js/pull/966/commits/d201e17579004f8a3a24357da51f2d5b2317ccad

Reply:
Because we changed the name of this test, I reverted the changes to `handleRewriteFrame` and the other test.